cache/util/getRetryStrategy.js

```diff
@@ -0,0 +1,12 @@
+// simple retry strategy
+
+// default max delay in milliseconds
+const DEFAULT_MAX_DELAY = 10000
+
+module.exports = function getRetryStrategy (maxDelay) {
+  const maximumDelay = maxDelay || DEFAULT_MAX_DELAY
+
+  return function (retryCount) {
+    return Math.min(1000 + (retryCount * 1000), maximumDelay)
```

Comment:
I know you were trying to keep this simple for now, but maybe add an element of randomness (jitter). Parallel retries will all happen at the same time if the delays are all the same.